components/Nav/NavMobile/NavMobile.js

```diff
@@ -34,17 +34,13 @@ export default class NavMobile extends Component {
     return (
       <header className={styles.NavMobile}>
         <Link href="/" prefetch={false}>
-          <button
-            className={classNames(styles.button, styles.logoButton)}
-            type="button"
-            name="dropdown"
-          >
+          <a aria-label="Go Home" className={classNames(styles.button, styles.logoButton)}>
```

Comment:
You know what... instead of `aria-label`, let's use the ScreenReaderOnly component.